src/impact-dashboard/EpidemicModelContext.tsx

```diff
@@ -152,6 +152,8 @@ export function getLocaleDefaults(
     totalIncarcerated:
       dataSource.get(stateCode)?.get(countyName)?.totalIncarceratedPopulation ||
       0,
+    totalPrisonPopulation:
+      dataSource.get(stateCode)?.get(countyName)?.totalPrisonPopulation || 0,
```

Comment:
won't this always be 0, since `Col J` only has values for states but not counties?

Reply:
The default value for `countyName` in the params declaration is "Total". So in the dataSource object for the given stateCode, without a countyName given, it pulls from the `['Total']` key which is correct.

![Screen Shot 2020-05-01 at 10 18 20 AM](https://user-images.githubusercontent.com/5872651/80825406-2aa3fe80-8b95-11ea-8761-e5fedfbac858.png)